ckanext/unaids/plugin.py

```diff
@@ -208,8 +207,9 @@ def after_update(self, context, pkg_dict):
                     dataset_id=pkg_dict["id"],
                     recipient_org_id=org_to_allow_transfer_to[0],
                 )
-        if pkg_dict.get("validate_package") and not context.get("_dont_validate"):
-            logging.warning("VALIDATING ENTIRE PACKAGE")
+
+        if pkg_dict['id'] in self.resources_to_validate_package and not context.get("_dont_validate"):
+            del self.resources_to_validate_package[pkg_dict['id']]
```

Comment:
`self.resources_to_validate_package` is somethimes used with `resource[id]` and sometimes with `dataset[id]`. Could you explain how exactly `self.resources_to_validate_package` works?

Reply:
As you have said in your other comment, in CKAN 2.9 pkg_dict here might be a resource or a package.